frontend/libs/studio-pure-functions/src/functions/arrayUtils/arrayUtils.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Removes duplicates from an array.
+ * @param array The array of interest.
+ * @returns The array without duplicates.
+ */
+export const studioRemoveDuplicates = <T>(array: T[]): T[] => [...new Set(array)];
```

Comment:
Er ikke denne i bruk noe sted?

Reply:
Det gir mening, men jeg tenker det er essensielt å verifisere at man kan importere funksjonene slik man ønsker. Da har andre som skal legge til funksjoner her også et eksempel å gå etter når de skal ta dem i bruk.

Reply:
Jeg er for! Da blir eksport/import også mer ryddig. Godt forslag!

Reply:
Yes, jeg er med på den! Bra forslag! 😄